etc/afpd/afpstats.c

```diff
@@ -111,7 +193,9 @@ int afpstats_init(server_child_t *childs_in)
     GThread *thread;
 
     childs = childs_in;
+#if 0
     dbus_g_thread_init();
```

Comment:
@smcv Is there a GDBus equivalent to `dbus_g_thread_init()` or is it not needed anymore?

This is the one authoritative source I could find but it's very brief. https://dbus.freedesktop.org/doc/dbus-glib/dbus-glib-DBusGConnection.html#dbus-g-thread-init

Reply:
There is no equivalent in GDBus, just delete it. GDBus is always multi-threaded and thread-safe.

With modern versions of libdbus, `dbus_g_thread_init()` doesn't do anything anyway.

Reply:
Cheers, then I understand what the docs was trying to say.